UFS-DA/ush/build_UFSDA.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# build_UFSDA.sh
+# 1 - determine if on supported host
+# 2 - load modules
+# 3 - run ecbuild
+# 4 - build
+# 5 - optional, run unit tests
+
+set -ex
+
+cd ..
+pwd=$(pwd)
```

Comment:
Don't do that.
```
dir_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Reply:
Ok, but with the caveat/addition of /../ within the cd so that it captures the root directory and not the `ush` directory